### ISSUE TITLE
GMP doesn't start after production build

### ISSUE THREAD

**Issue by austin-whittington** [gemini-hlsw/gmp#3]
GMP was able to successfully build using `mvn install` and build the tar.gz package with `mvn -Pproduction clean install`. However, after building, when running `start.sh`, it throws a `org.ops4j.pax.runner.platform.PlatformException` for the `checker-compat-qual-2.0.0.jar` not being a valid bundle. Just to experiment, I got rid of that bundle and there were more that threw errors.

Platform - Centos 7
Java - openjdk 1.8.0_252
Maven - 3.0.5

**Comment by iarriagada** [comment]
We'll check this out, in the meantime, can you try running `mvn pax:run` to launch a local version of GMP directly from the base code?
Thanks

**Comment by austin-whittington** [comment]
Yes, that runs, and `lb` shows a list of GIAPI and GMP services. However, when running `help` in the prompt, none of the `gmp:*` commands show, and I can't check active statuses/names. Is that expected?

**Comment by iarriagada** [comment]
Not sure. Are you able to connect the icsw to the GMP? Is the `giapi-tester` connecting to GMP?

**Comment by cquiroz** [comment]
The issue with `checker-compat` is that is included as a transitive dependency but not OSGi compatible. We need to update the build to not include it in the distribution

**Comment by cquiroz** [comment]
The `gmp:*` commands sounds like again we are missing something on the list of command bundles. During the last large update (to java 1.8) some of these were probably missing.
An any rate, I'd still recommend `giapi-tester` to get status names, etc
